terraform/README.md

```diff
@@ -0,0 +1,65 @@
+# Grafana-agent-k8s Terraform Module
+
+This Grafana-agent-k8s Terraform module aims to deploy the [grafana-agent-k8s charm](https://charmhub.io/grafana-agent-k8s) via Terraform.
```

Comment:
How about we mention that the module uses Juju Terraform provider, as we did for the SD-Core modules?

Reply:
Done